src/Build/BackEnd/Components/RequestBuilder/TargetBuilder.cs

```diff
@@ -766,5 +767,45 @@ private async Task<bool> CompleteOutstandingActiveRequests(string targetName)
 
             return false;
         }
+
+        private void ComputeAfterTargetFailures(string[] targetNames)
+        {
```

Comment:
could `targetNames` be null?

Reply:
No, it is treated as non-null by the caller and since this method is a private helper I don't think arguments need to be validated. Looking at other helpers in this file, for example `PushTargets`, I don't see such a validation either.

Reply:
Agreed. We also need to develop a "what do we do for `#nullable`" strategy for MSBuild. Right now I've only explicitly enabled it in a few new files. We need to decide whether we want to go with the "turn it on as you touch stuff" or the "turn it off in every file" strategy. Here, no action needed.